media/css/firefox/mobile.scss

```diff
@@ -844,7 +844,7 @@ html[dir="rtl"] {
         }
 
         &.feature-block:before {
-            background-position: right top -4px;
+            background-position: left top -10px;
```

Comment:
This fixes the icon bleed for right-to-left languages, but the icon is no longer aligned correctly:

<img width="800" alt="Screenshot 2019-09-16 at 10 50 27" src="https://user-images.githubusercontent.com/400117/64948855-e1a27280-d86f-11e9-842e-bb622bb5cc15.png">

I'd suggest changing this to:

```suggestion
            background-position: right top -10px;
```